crates/net/eth-wire/src/hello.rs

```diff
@@ -99,8 +99,9 @@ impl HelloMessageBuilder {
         HelloMessage {
             protocol_version: protocol_version.unwrap_or_default(),
             client_version: client_version.unwrap_or_else(|| DEFAULT_CLIENT_VERSION.to_string()),
-            capabilities: capabilities
-                .unwrap_or_else(|| vec![EthVersion::Eth66.into(), EthVersion::Eth67.into()]),
+            capabilities: capabilities.unwrap_or_else(|| {
+                vec![EthVersion::Eth66.into(), EthVersion::Eth67.into(), EthVersion::Eth68.into()]
```

Comment:
I don't think we should enable this by default already?

Reply:
agreed, this is a future upgrade, `67` is still considered the current version

Reply:
agreed. I thought it's safe because it's still including `66`, `67`, but I was too naive and aggressive. Definitely, I agree not to enable it yet and need to verify it.

Reply:
follow it up with 651fc4ddee1565cfb935695b8ac225e3b8321edb